app/survey_config/business_config.py

```diff
@@ -101,6 +85,38 @@ def get_service_links(
 
         return links
 
+    def get_footer_links(self, cookie_has_theme: bool) -> list[dict]:
+
+        if cookie_has_theme:
+            return [
+                Link(lazy_gettext("What we do"), self.what_we_do_url).__dict__,
+                Link(lazy_gettext("Contact us"), self.contact_us_url).__dict__,
+                Link(
+                    lazy_gettext("Accessibility"),
+                    self.accessibility_url,
+                ).__dict__,
+            ]
+
+        return [
+            Link(lazy_gettext("What we do"), self.what_we_do_url).__dict__,
+            Link(
+                lazy_gettext("Accessibility"),
+                self.accessibility_url,
+            ).__dict__,
+        ]
```

Comment:
Good to avoid the duplication. Applies to the other suvery configs also.
```suggestion
    def get_footer_links(self, cookie_has_theme: bool) -> list[dict]:
        links = [Link(lazy_gettext("What we do"), self.what_we_do_url).__dict__]

        if cookie_has_theme:
            links.append(Link(lazy_gettext("Contact us"), self.contact_us_url).__dict__)

        links.append(
            Link(
                lazy_gettext("Accessibility"),
                self.accessibility_url,
            ).__dict__
        )

        return links
```

Reply:
Removed duplication from all configs.